doc/releases/release-notes-4.2.rst

```diff
@@ -26,6 +26,53 @@
 
 .. _zephyr_4.2:
 
+.. _zephyr_4.2.1:
+
+Zephyr 4.2.1
+############
+
+This is a bugfix release for Zephyr 4.2.0. The following issues are addressed by this release:
+
+:github:`92393` LIS2DUX12 setting incorrect ODR configuration
+:github:`93011` stm32u0: interrupt vectors for lpuart1 and lpuart2 are wrong in device tree
+:github:`93220` Twister tests with coverage enabled crash during startup
+:github:`93265` Stm32u5x: clock disable in sleep mode fix causing USB HS device initialization stalls
+:github:`93342` doc: Documentation issue in 'connectivity/networking/api/tftp'
+:github:`93350` net: tcp: keep-alive "pings" are no longer acknowledged with v4.2.0
+:github:`93364` tests/subsys/llext fails after tagging 4.2
+:github:`93424` Missing break after IPV6_MULTICAST_IF option processing
+:github:`93445` STM32U535/545: dtc error for node wkup-pin@8 referencing non-existent port gpiof
+:github:`93519` drivers: nxp_lpspi: Large performance degradation
+:github:`93594` Nicla Vision has bad I2C address for VL53L1X sensor
+:github:`93611` doc: Documentation issue in 'releases/release-notes-4.2'
+:github:`93790` drivers: console: posix_arch_console: build warning with ``CONFIG_PRINTK=n``
+:github:`93792` USB samples not working on ``max32690evkit``
+:github:`93831` drivers: can: nrf: fix invalid pointer leading to undef behavior
+:github:`93839` I3C: STM32: Has a Mutex deadlock on DAA Failure
+:github:`94000` EXT2 fs_stat on mount point fails with a NULL pointer dereference.
+:github:`94047` Shell device name tab completion for regulator parents show incorrect names
+:github:`94063` SPI loopback tests failing on MAX32 SoCs.
+:github:`94083` stm32:  u3:  low clock frequency for the RNG periphera
+:github:`94093` Build error when OCRAM node is disabled in device tree (status = "disabled") [NXP RT1061]
+:github:`94107` ppp link-layer address configuration is wrong
+:github:`94325` iar: incorrect assembler optimzation flag
+:github:`94702` Locally Administered Address (LAA) bit missing for STM32 ethernet driver
+:github:`94703` Locally Administered Address (LAA) bit set on wrong index for NXP i.MX ethernet driver
+:github:`94774` modules/picolibc: Roll back unnecessary changes, synchronize with SDK 0.17.4
+:github:`94842` iar: Zephyr 4.2 fails with IAR EWARM 9.70.1
+:github:`94954` Spurious error log "No valid legacy adv to resume" in bt_hci_le_enh_conn_complete when using bt_le_ext_adv_start()
+:github:`95146` [Backport v4.2] i2c: stm32: non-RTIO driver fixes on PM and on STM32F7 from Zephyr v4.3
```

Comment:
I'm pretty sure there were more backports. Do we want to remove that prefix from this line specifically?  Likely also line 68, 72, and 73

Reply:
Yes, definitely remove backport prefixes.

Reply:
> I'm pretty sure there were more backports

I think this covers it, but I could be wrong- pulled these from the github API using the PR description, and targeting merged PRs against the 4.2 branch, of which there are 38 (and there are 38 entries here). Happy to post the script I used though

> Yes, definitely remove backport prefixes.

Updated to remove them.